mono/mini/mini-llvm.c

```diff
@@ -6345,7 +6345,7 @@ process_bb (EmitContext *ctx, MonoBasicBlock *bb)
 			args [1] = LLVMConstInt (LLVMInt1Type (), 1, FALSE);
 			cmp = LLVMBuildCall (ctx->builder, get_intrins (ctx, INTRINS_EXPECT_I1), args, 2, "");
 
-			LLVMBuildCondBr (builder, cmp, cont_bb, poll_bb);
+			mono_llvm_build_weighted_branch (builder, cmp, cont_bb, poll_bb, 64, 4);
```

Comment:
why 64 and 4?  is there a documentation for that somewhere?

Reply:
@imhameed ^^

Reply:
I agree, make them constants at least

Reply:
In my sample https://godbolt.org/z/IS7Qqk `-lower-expect` used `1` and `2000` weights for branches.

Reply:
and I am actually not sure we use weights here in a correct order ("4" should be applied for the "we need to call state_pool" branch

Reply:
cmp is the result of expect, so its expected to be 1.